rlclientlib/utility/config_utility.cc

```diff
@@ -110,7 +110,13 @@ namespace reinforcement_learning { namespace utility { namespace config {
       { "InitialCommandLine"        , name::MODEL_VW_INITIAL_COMMAND_LINE},
       { "initialCommandLine"        , name::MODEL_VW_INITIAL_COMMAND_LINE},
       { "ProtocolVersion"           , name::PROTOCOL_VERSION},
-      { "protocolVersion"           , name::PROTOCOL_VERSION}
+      { "protocolVersion"           , name::PROTOCOL_VERSION},
```

Comment:
This is a legacy mapping. Why are we adding more values to this? It is only intended for compat. New keys should only ever be used in a single form with no need for mapping

Reply:
Fixed.